nuxeo-business-days-management-core/src/main/java/org/nuxeo/business/days/management/checker/CSVHolidaysChecker.java

```diff
@@ -28,15 +28,13 @@
 import java.io.InputStreamReader;
 import java.io.Reader;
 import java.text.ParseException;
-import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.HashSet;
-import java.util.Iterator;
```

Comment:
In the future please do cleanups in separate commits. If it's your IDE doing this automatically, please deactivate it.